s3-native-client/build.gradle.kts

```diff
@@ -0,0 +1,32 @@
+
+plugins {
+    id("software.amazon.smithy").version("0.5.0")
+    id("com.github.johnrengelman.shadow") version "6.1.0"
+    `java-library`
+}
+
+repositories {
+    mavenLocal()
+    mavenCentral()
+}
+
+tasks.compileJava {
+    dependsOn("smithyBuildJar")
+}
+
+sourceSets {
+    main {
+        java {
+            //srcDirs("${buildDir}/smithyprojections/${project.name}/source/crt")
```

Comment:
This has relevance to come back to life for the subproject to compile generated code + handwritten code together. I can remove it, but think it can help someone else more easily get to that better end state more quickly.